pkg/vault/test/foundry/unit/VaultUnitLiquidity.t.sol

```diff
@@ -807,7 +807,7 @@ contract VaultUnitLiquidityTest is BaseTest, VaultContractsDeployer {
 
         for (uint256 i = 0; i < tokens.length; i++) {
             poolData.tokens[i] = tokens[i];
-            poolData.decimalScalingFactors[i] = 1e18;
+            poolData.decimalScalingFactors[i] = 1; // 18 decimals
```

Comment:
```suggestion
            poolData.decimalScalingFactors[i] = 1; // 18 decimals, so 10^(18-18) = 1
```

I'd explain it a bit better, just to have in the whole code that the scaling factor is 10^(18-tokenDecimals)